packages/native-core/src/layout_attributes.rs

```diff
@@ -19,573 +19,400 @@
+                        distribution,
+                    ) => match distribution {
+                        lightningcss::properties::align::ContentDistribution::SpaceBetween => {
+                            SpaceBetween
+                        }
+                        lightningcss::properties::align::ContentDistribution::SpaceAround => {
+                            SpaceAround
+                        }
+                        lightningcss::properties::align::ContentDistribution::SpaceEvenly => {
+                            SpaceEvenly
+                        }
+                        lightningcss::properties::align::ContentDistribution::Stretch => Stretch,
+                    },
+                    lightningcss::properties::align::AlignContent::ContentPosition(_, position) => {
+                        match position {
+                            lightningcss::properties::align::ContentPosition::Center => Center,
+                            lightningcss::properties::align::ContentPosition::Start => todo!(),
+                            lightningcss::properties::align::ContentPosition::End => todo!(),
+                            lightningcss::properties::align::ContentPosition::FlexStart => {
+                                FlexStart
+                            }
+                            lightningcss::properties::align::ContentPosition::FlexEnd => FlexEnd,
+                        }
+                    }
+                };
+            }
+            Property::JustifyContent(justify, _) => {
+                use JustifyContent::*;
+                style.justify_content = match justify {
+                    lightningcss::properties::align::JustifyContent::Normal => todo!(),
+                    lightningcss::properties::align::JustifyContent::ContentDistribution(
+                        distribution,
+                    ) => match distribution {
+                        lightningcss::properties::align::ContentDistribution::SpaceBetween => {
+                            SpaceBetween
+                        }
+                        lightningcss::properties::align::ContentDistribution::SpaceAround => {
+                            SpaceAround
+                        }
+                        lightningcss::properties::align::ContentDistribution::SpaceEvenly => {
+                            SpaceEvenly
+                        }
+                        lightningcss::properties::align::ContentDistribution::Stretch => todo!(),
+                    },
+                    lightningcss::properties::align::JustifyContent::ContentPosition(
+                        _,
+                        position,
+                    ) => match position {
+                        lightningcss::properties::align::ContentPosition::Center => Center,
+                        lightningcss::properties::align::ContentPosition::Start => todo!(),
+                        lightningcss::properties::align::ContentPosition::End => todo!(),
```

Comment:
`Start` and `End` can be mapped to `FlexStart` and `FlexEnd`. They're the same in practice. Taffy 0.3 renamed these variants in Taffy to `Start` and `End`. Same applies for the other align/justify properties. Note: this does apply to `SelfStart` and `SelfEnd`.

Reply:
It appears there is some difference at least in the case of justify-content when flex-direction is row-reverse or column-reverse (see https://codepen.io/rachelandrew/pen/KKmazZR)

Reply:
Ah, so *that's* the difference (the \*-reverse flex directions). I'll create an issue in the Taffy repo to support both. The 0.3 previews has renamed the variants to `Start`/`End`, but is still implementing the existing `FlexStart`/`FlexEnd`  behaviour.